docs/LibraryEvolution.rst

```diff
@@ -607,9 +581,9 @@ stored subscripts. This means that the following changes are permitted:
 - Changing or removing a default argument is a `binary-compatible
   source-breaking change`.
 
-Like properties, subscripts can be marked ``@inlineable``, which makes changing
-the body of an accessor a `binary-compatible source-breaking change`. Any
-inlineable accessors must follow the rules for `inlineable functions`_, as
+Like properties, subscripts can be marked ``@inlinableAccess``, which makes
+changing the body of an accessor a `binary-compatible source-breaking change`.
```

Comment:
Why is it source breaking? What if I just optimize an algorithm inside my accessor body?

Reply:
It's not source-incompatible, sure. Maybe we need a different category name for "binary-compatible, but still has lasting effects across versions".